src/inline.jl

```diff
@@ -1,6 +1,9 @@
 import Base: display, redisplay
 
-immutable InlineDisplay <: Display end
+@static if !isdefined(Base, :AbstractDisplay) # remove when Compat.jl#482 is merged and tagged
```

Comment:
Shouldn't this be:
```jl
if !isdefined(Base, :AbstractDisplay) && !isdefined(Compat, :AbstractDisplay)
  ...
```
(No need for `@static` in global scope unless the expressions have `ccall` or macros...)

Reply:
Actually, this if statement can be deleted completely as the `AbstractDisplay` definition was tagged in Compat v0.52.0